src/platforms/generic_unix/sys.c

```diff
@@ -199,6 +199,7 @@ void sys_consume_pending_events(GlobalContext *glb)
         if (listener_fd >= 0) {
             fds_count++;
         }
```

Comment:
I'm not sure about the reason of this change. I need to think about this.

Reply:
Without advancing the listener point, the loop does not terminate.